rs/bitcoin/kyt/BUILD.bazel

```diff
@@ -4,47 +4,27 @@ load("//bazel:defs.bzl", "rust_ic_test")
 
 package(default_visibility = ["//visibility:public"])
 
+LIB_SRCS = [
+    "src/blocklist.rs",
+    "src/lib.rs",
+    "src/types.rs",
+]
+
 rust_library(
     name = "btc_kyt_lib",
-    srcs = glob(
-        ["src/**"],
-        exclude = ["src/main.rs"],
-    ),
-    compile_data = [
-        "templates/dashboard.html",
-    ],
+    srcs = LIB_SRCS,
```

Comment:
I thought about that, but the dashboard needs access to internal state, which I don't think is appropriate to expose on a library's interface. Another reason is that I think it is better to move anything `thread_local!` out of a library so that the library remains stateless. I hope it makes sense.

Reply:
I see, 

> Another reason is that I think it is better to move anything thread_local! out of a library so that the library remains stateless

I think that's a good argument! Would it be possible to have the library as a separate crate? Unless someone is actively looking at the Bazel file, it's otherwise very hard to know what's in the library and what's not in it. Also ok to do this in a separate PR if that's easier. (This would also allow Cargo to know about the library crate).

Reply:
Will do it in a separate PR. Created a ticket to track this https://dfinity.atlassian.net/browse/XC-81